src/impact-dashboard/CurveChart.tsx

```diff
@@ -44,51 +55,27 @@ const ChartContainer = styled.div`
   }
 `;
 
-export default function CurveChart({ curveData, hospitalBeds }) {
-  const lines = Object.entries(curveData).map(([bucket, values]) => ({
-    title: bucket,
-    key: bucket,
-    coordinates: values.map((count, index) => ({
-      count,
-      days: index + 1,
-    })),
-  }));
-
-  // make sure the Y axis includes hospital beds threshold
-  const maxCurvePeak = lines.reduce(
-    (highestPeak, { coordinates }) =>
-      Math.max(
-        highestPeak,
-        coordinates.reduce(
-          (linePeak, { count }) => Math.max(linePeak, count),
-          0,
-        ),
-      ),
-    0,
-  );
-  const yMax = Math.ceil(Math.max(hospitalBeds, maxCurvePeak) / 1000) * 1000;
-
-  // TODO: factor colors out to variables somewhere?
-  // also these are not totally final
-  const colors = {
-    exposed: "#00413e",
-    infectious: "#de5558",
-    hospitalized: "#65d4ff",
-    hospitalBeds: "#de5558",
-  };
+export default function CurveChart({ curveData, hospitalBeds, markColors }) {
```

Comment:
If possible, can you TypeScript the props?

```tsx

interface Props {
  curveData: [string, number[]][],
  hospitalBeds: number,
  markColors: MarkColors,
}

const CurveChart: React.FC<Props> = ({ curveData, hospitalBeds, markColors }) => {
   ...
}

export default CurveChart
```

This makes it easier to see how to use the component without needing to look at the implementation.